### PR TITLE
ENH simulator looking more brain like

### PR DESCRIPTION
**Pull request by pushtheworldllc**
Simulator is looking more brainy then it ever has!

### CODE REVIEW

**Comment by alexcastillo** [comment]
This is looking great. Can we merge it @pushtheworldllc ?

**Comment by pushtheworldllc** [comment]
Did you review the code? Just need someone to +1 it

**Comment by alexcastillo** [comment]
Yes, everything looks great. Go for it!

File: README.md [hunk 1]
```diff
@@ -288,6 +288,11 @@ Board optional configurations.
 * `verbose` To output more messages to the command line.
 * `simulate` Full functionality, just synthetic data.
 * `simulatorSampleRate` - The sample rate to use for the simulator (Default is `250`)
+* `simulatorAlpha` - {Boolean} - Inject and 10Hz alpha wave in Channels 1 and 2 (Default `true`)
+* `simulatorLineNoise` - Injects line noise on channels.
+  * `60Hz` - 60Hz line noise (Default) (ex. __Unite States__)
```
Comment (teonbrooks) [inline README.md]: Unite --> United

File: openBCISample.js [hunk 2]
```diff
@@ -278,56 +276,79 @@ var sampleModule = {
...
-        return function(previousSampleNumber) {
-            var newSample = self.newSample();
+        // Init arrays to hold coefficients for each channel
+        var b0 = new Array(numberOfChannels);
+        var b1 = new Array(numberOfChannels);
+        var b2 = new Array(numberOfChannels);
 
-            //console.log('New Sample: ' + JSON.stringify(newSample));
+        // Init coefficients to 0
+        b0.fill(0);
```
Comment (teonbrooks) [inline openBCISample.js]: You could bundle this with the Array init above if you want

File: openBCISample.js [hunk 3]
```diff
@@ -278,56 +276,79 @@ var sampleModule = {
...
-                        if (sinePhaseRad[i] > 2 * pi) {
-                            sinePhaseRad[i] -= 2 * pi;
-                        }
-                        newSample.channelData[i] += (10 * sqrt2 * Math.sin(sinePhaseRad[i]))/uVolts;
-                        break;
-                    case 3:
-                        sinePhaseRad[i] += 2 * pi * sineWaveFreqHz50 / sampleRateHz;
-                        if (sinePhaseRad[i] > 2 * pi) {
-                            sinePhaseRad[i] -= 2 * pi;
+                    case 0: // Add 10Hz signal to channel 1... briany
```
Comment (teonbrooks) [inline openBCISample.js]: briany --> brainy

File: openBCISample.js (hunk 2 above)
Comment (pushtheworldllc) [inline openBCISample.js]: how?

Comment (teonbrooks) [inline openBCISample.js]: by chaining, `Array(numberOfChannels).fill(0);`

**Pull request merged**